lint-configs/.clang-format

```
@@ -60,6 +60,7 @@ BreakBeforeTernaryOperators: true
 BreakConstructorInitializers: "BeforeColon"
```

```suggestion
BreakConstructorInitializers: "BeforeColon"
BreakFunctionDefinitionParameters: false
```

This is a new option that's unfortunately omitted from the release notes.